### DISCUSSION TITLE
Set-PNPListItem SystemUpdate parameter missing

### DISCUSSION THREAD

**Discussion by Markus-F** [pnp/powershell#1551] [Q&A]
Dear all,

I am trying to update the content type of list items via PNP in a SharePoint 2016 environment. Unfortunately the SystemUpdate parameter is missing. Version used is 3.29.2101.0. Is the parameter only available with SharePoint Online? If yes, is there another way to update an item on-premise with PNP without changing the modified and modified by values?

Best regards,
Markus

**Comment by LeonArmston** [comment]
Hi @Markus-F 

The new version of PnP PowerShell is the repository that you are in and it is only for SharePoint online. The PnP PowerShell modules for on-premise are no longer maintained (since around December 2020).

As it's on-prem I would update the list item not via PnP but via the Client Side Object Model which does have a -SystemUpdate parameter

This script should do it that I found on a blog: [SharePoint 2013: Change The Content Type Using Powershell Script](https://www.c-sharpcorner.com/article/sharepoint-2013-change-the-content-type-using-powershell-scr/) - you would run this on your server

Thanks

Leon

**Reply by Markus-F** [reply]
Thanks @LeonArmston . I solved it via CSOM.
